README.md

```diff
@@ -1094,6 +1095,10 @@ snapshot. Set it multiple times for multiple ignore paths.
 Set this flag to the number of retries that should happen for the extracting an
 image filesystem. Defaults to `0`.
 
+#### Flag `--image-download-retry`
+
+Set this flag to the number of retries that should happen when downloading the remote image. Defaults to `0`.
```

Comment:
nit: can you add the default time in between the retries to help users understand that n=5 -> 5 seconds of retries